packages/back-end/src/scripts/migrate-encryption-key.ts

```diff
@@ -35,6 +36,7 @@ async function run() {
   const allDatasources = await _dangerousGetAllDatasources();
   for (let i = 0; i < allDatasources.length; i++) {
     const ds = allDatasources[i];
+    const context = await getContextForAgendaJobByOrgId(ds.organization);
```

Comment:
In a separate PR, I'm going to update the name of this as it's no longer just being used by Agenda jobs. Thinking something like `dangerouslyGetContextObj`.

Reply:
Makes sense. Another naming suggestion `getFullAccessContextObj`